.gitignore

```diff
@@ -5,3 +5,4 @@ __pycache__
 *.egg-info/
 /build/
 /dist/
+.DS_Store
```

Comment:
Please don't add such files to the project .gitignore. Instead, you should add them to your global .gitignore:
https://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file#5724484
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer
https://git-scm.com/docs/gitignore